trailblazer/dto/analyses_request.py

```diff
@@ -18,3 +18,4 @@ class AnalysesRequest(BaseModel):
     priority: list[TrailblazerPriority] | None = []
     type: list[TrailblazerTypes] | None = []
     comment: list[str] | None = []
+    order_id: int | None = Field(alias="orderId", default=None)
```

Comment:
In the end, all analyses should be associated with an order - but we'll have to fix that separately I guess

Reply:
Yes, we want these to have an order and for this field to not be nullable. But that is out of scope for now.